### PR TITLE
Add rental cancelation policies

### PR DESCRIPTION
**Pull request by ZenCocoon**

### CODE REVIEW

**Comment by ZenCocoon** [comment]
WARNING: The `body_translations` will certainly be renamed `body` by the time this get's marked as stable.

**Comment by ZenCocoon** [comment]
@Azdaroth can you please review when time.

**Comment by Azdaroth** [comment]
:+1:

**Comment by ZenCocoon** [comment]
@Azdaroth I think we must release this as stable and look for the renaming of the body in v4 instead. If you agree, can you please make sure it gets merged and deployed. Thanks.

File: responses/rental_cancelation_policies/public-scope.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  },
+  "rental_cancelation_policies": [
+    {
+      "links": {
+        "rental": 1,
+        "rental_cancelation_policy_items": [
+          1,
+          2
+        ]
+      },
```
Comment (StoneFrog) [inline responses/rental_cancelation_policies/public-scope.json]: it returns also `body` which seems to be the body translation in default locale, so sth like
`"body": "CANCELLATIONS. \nAny cancellation must be sent by registered letter, on strength of postmark."`
should be added

File: content/reference/endpoints/rental_cancelation_policies.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  <li class="disabled"><a>OAuth Scopes:</a></li>
+  <li class="active"><a href="#public" role="tab" data-toggle="pill">public</a></li>
+</ul>
+<div class="tab-content" markdown="1">
+  <div class="tab-pane active" id="public" markdown="1">
+Name             | Type    | Read/Write | Description
+-----------------|---------|------------|------------
+id               | Integer | Read       | Rental cancelation policy's id.
+-----------------|---------|------------|------------
+body_translations | [Object](/reference/enums#formats)     | Read       | Rental cancelation policy's body, list of locales described in [enums section](/reference/enums#locales).
```
Comment (StoneFrog) [inline content/reference/endpoints/rental_cancelation_policies.md]: missing `body` attribute as well - seems to be body translation in default locale

File: content/reference/changelog.md [hunk 3]
```diff
@@ -1,5 +1,7 @@
 # Changelog
 
+  * [api improvement] Add `rental_cancelation_policies` and `rental_cancelation_policy_items` endpoints.
```
Comment (StoneFrog) [inline content/reference/changelog.md]: no need for updating changelog any longer, will also resolve conflicts

**Comment by Azdaroth** [comment]
@StoneFrog Sadly, I don't think Seb will be able to make those fixes :/. Can you take over please?

**Comment by StoneFrog** [comment]
Sure

**Comment by StoneFrog** [comment]
@Azdaroth could you check my updates? Don't feel like self-approving

**Pull request merged**